### PR TITLE
Add GitHub Action test runner

### LINKED ISSUE #9: Add CI with GitHub actions
To test the package

### PR DESCRIPTION
**Pull request by evayde**
Simple runner. Resolves #9

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: NPM Test
+on: [push]
```
Comment (neelkamath) [inline .github/workflows/test.yml:2]: This should probably also run on `pull_request` for contributors.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Set up NodeV14
```
Comment (neelkamath) [inline .github/workflows/test.yml:11]: This step can probably be left out as you're not testing on a matrix, and GitHub Actions already has a recent node.js version preinstalled. I don't think we've explicitly documented which exact version of node.js is needed to run the tests.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+      - run: npm i
```
Comment (neelkamath) [inline .github/workflows/test.yml:15]: You should prefer `npm cit` over `npm i` + `npm t`.

**Comment by evayde** [comment]
Thank you for your input @neelkamath

**Pull request merged**